cypress/page/shoping-cart.page.ts

```diff
@@ -5,7 +5,7 @@ class ShopingCartPage {
       this.checkbtn = ".cart_navigation span";
```

Comment:
you can improve this one, try not to depend on span

Reply:
@kliver98  donde all